Upstream: Should be
Reason: Fixes compilation under gcc-10 which has -fno-common

diff --git a/src/stats.c b/src/stats.c
index 35dbf7d..76bfbee 100644
--- a/src/stats.c
+++ b/src/stats.c
@@ -82,7 +82,7 @@ struct statsdata {
     char compiler[MAX_FILENAME_LEN];
 };
 
-const char *stats_text[20] = { "TCP_ACCEPT", "REJ_BAD_REQ", "REJ_OVERLOAD",
+extern const char *stats_text[20] = { "TCP_ACCEPT", "REJ_BAD_REQ", "REJ_OVERLOAD",
     "COMPILE_OK", "COMPILE_ERROR", "COMPILE_TIMEOUT", "CLI_DISCONN",
     "OTHER" };
 
diff --git a/src/stats.h b/src/stats.h
index 9bde285..74d4690 100644
--- a/src/stats.h
+++ b/src/stats.h
@@ -33,7 +33,7 @@ enum stats_e { STATS_TCP_ACCEPT, STATS_REJ_BAD_REQ, STATS_REJ_OVERLOAD,
                 STATS_COMPILE_OK, STATS_COMPILE_ERROR, STATS_COMPILE_TIMEOUT,
                 STATS_CLI_DISCONN, STATS_OTHER, STATS_ENUM_MAX };
 
-const char *stats_text[20];
+extern const char *stats_text[20];
 
 int  dcc_stats_init(void);
 void dcc_stats_init_kid(void);
